keel-core/src/main/kotlin/com/netflix/spinnaker/keel/veto/unhappy/UnhappyVeto.kt

```diff
@@ -44,7 +44,8 @@ class UnhappyVeto(
   private val diffFingerprintRepository: DiffFingerprintRepository,
   private val unhappyVetoRepository: UnhappyVetoRepository,
   private val dynamicConfigService: DynamicConfigService,
-  @Value("veto.unhappy.waiting-time")
+
+  @Value("\${veto.unhappy.waiting-time:PT10M}") // tests fail if default not specified here
   private val configuredWaitingTime: String = "PT10M"
```

Comment:
You should be able to remove this PT10M now that the Value annotation has a default value.

Reply:
I was going to write, "I can't, because the constructor is invoked directly in the code without  configuredWaitingTime as an arugment", but that only happens in test code, where there's no harm in modifying the test to pass the argument explicitly. I'll update the PR.

Reply:
Addressed in f021ed6

Reply:
I mean, not necessarily even as an automated test, just to confirm the behavior. If it's an automated test all the better, sure.